guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,395 @@ Illustration sources:
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-progress@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-progress@2x.png"
+   layout = "shadow"
+   caption = "User enters their recovery phrase. The app notifies them if they have typed a word incorrectly."
+   alt-text = "Screen showing an incorrectly typed word highlighed in red."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-complete.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-complete@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-manual-recovery-phrase-complete@2x.png"
+   layout = "shadow"
+   caption = "User successfully enters their entire recovery phrase."
+   alt-text = "Screen showing a 12 word recovery phrase in green text"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restoring@2x.png"
+   layout = "shadow"
+   caption = "App begins to restore, pulling the channels backup automatically from the cloud."
+   alt-text = "Screen showing an animation as the wallet is being restored"
+   width = 250
+   height = 541
+%}
 
-As outlined earlier in the chapter, we consider an [automatic cloud backup](/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/) a good option, especially for beginners. One of the major UX benefits of this is a seamless recovery process. Assuming they are restoring in the same wallet application originally used, all the user needs to do to gain access to their funds again is to log in to their cloud storage account (typically iCloud or Google Drive).
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored.png"
+   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-restored@2x.png"
+   layout = "shadow"
+   caption = "Wallet is finished being restored."
+   alt-text = "Screen showing that the wallet has been restored."
+   width = 250
+   height = 541
+%}
 
-### Restore manually with a recovery phrase
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restore-finish.png"
```

Comment:
This screen uses the old home screen design that shows the balance. Maybe show the activity screen that shows balance and transactions - as a sign that it was all restored?

Reply:
Done